packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement, useContext, useEffect } from 'react';
+import classNames from 'classnames';
+import { Button } from './buttons/Button';
+import { ScrollOnboardingVersion } from '../lib/featureValues';
+import { cloudinary } from '../lib/image';
+import AnalyticsContext from '../contexts/AnalyticsContext';
+import { AnalyticsEvent, TargetType } from '../lib/analytics';
+
+const GaussianBlur = (): ReactElement => {
+  return (
+    <div className="absolute w-52 h-52 rounded-full opacity-32 blur-2xl bg-theme-color-cabbage" />
+  );
+};
+
+const versionToContainerClassName: Record<ScrollOnboardingVersion, string> = {
+  [ScrollOnboardingVersion.V1]: 'h-60',
+  [ScrollOnboardingVersion.V2]: 'h-[36rem]',
+};
+const versionToButtonClassName: Record<ScrollOnboardingVersion, string> = {
+  [ScrollOnboardingVersion.V1]: 'w-[16.25rem]',
+  [ScrollOnboardingVersion.V2]: 'w-40',
+};
+const versionToButtonText: Record<ScrollOnboardingVersion, string> = {
+  [ScrollOnboardingVersion.V1]: 'Customize your feed',
+  [ScrollOnboardingVersion.V2]: 'Start',
+};
+
+interface ScrollFeedFiltersOnboardingProps {
+  version: ScrollOnboardingVersion;
+  onInitializeOnboarding: () => void;
+}
+export default function scrollFeedFiltersOnboarding({
+  version,
+  onInitializeOnboarding,
+}: ScrollFeedFiltersOnboardingProps): ReactElement {
+  const { trackEvent } = useContext(AnalyticsContext);
+  useEffect(() => {
+    trackEvent({
+      event_name: AnalyticsEvent.Impression,
+      target_type: TargetType.ScrollBlock,
+      target_id: version,
+    });
+  }, []);
+
+  return (
+    <div
+      className={classNames(
+        'flex fixed bottom-0 left-60 z-3 gap-16 justify-center items-center w-full pr-40',
+        versionToContainerClassName[version],
+      )}
+      style={{
```

Comment:
There seems to be an issue with the background color. It should not have a hard stop
<img width="823" alt="image" src="https://user-images.githubusercontent.com/1488164/208103922-cb81cb09-0fa8-4faf-a9e0-7f8cd779292f.png">